src/tests/unit/services/YotiService.test.ts

```diff
@@ -233,10 +232,16 @@ describe("YotiService", () => {
 
 			axiosMock.post.mockResolvedValue({ data: { session_id: "session123" } });
 
+			const sessionDeadlineDate = new Date(new Date().getTime() + yotiService.YOTI_SESSION_TTL_DAYS * 24 * 60 * 60 * 1000);
+			sessionDeadlineDate.setHours(22, 0, 0, 0);
+
 			const sessionId = await yotiService.createSession(personDetails, selectedDocument, "GBR", YOTICALLBACKURL);
 
 			expect(generateYotiRequestMock).toHaveBeenCalled();
-			expect(axios.post).toHaveBeenCalledWith("https://example.com/api/sessions", createSessionPayload, {});
+			expect(axios.post).toHaveBeenCalledWith("https://example.com/api/sessions", {
```

Comment:
should we have separate tests? "it should create a session", "it should set session_deadline", "it should set resources_ttl" etc. ?

Reply:
Any reason for that? Only cause we're testing the full createSession payload in this test and it feels like repeated logic to check different parts of the same payload

Reply:
we're dynamically constructing the expected response in lines 235 and 236, so we're testing that specific logic  in the createSession service, it may be worth separating that from the rest of the test which is just checking against a static payload. Also, it occurred to me, and it is an edge case, this test might fail if you run it exactly on midnight, since the expected answer may then be the day before the actual answer - to be bulletproof here we should useFakeTimers perhaps?

Reply:
Done :)